docs/examples/debug.md

```diff
@@ -1,11 +1,11 @@
 # Potentially helpful debug commands
 
-## watch datamover backup resources live
+## watch DataMover backup resources live
 ```
-watch -n 10 ./datamover_resources.sh -b -d 
+watch -n 10 go run getDataMoverResources.go -b -d
```

Comment:
btw `go run` is just a quick and dirty way to run from go code directly.

For a published code, it would be good to `go build getDataMoverResources.go -o getDataMoverResources` then use compiled binary for faster speed.
```
./getDataMoverResources
```

Reply:
oh yeah, forget that, I'm used to Python ehhehe Going to add in the doc to user to compile first (and also tell about flag to switch between 1.2 and 1.3 datamover)